src/views/conference/2020/index/l10n.json

```
@@ -1,26 +1,26 @@
 {
-    "conference-2020.title": "Scratch Conference 2020:",
+    "conference-2020.title": "Scratch Conference 2021:",
     "conference-2020.subtitle": "Let's Create Together!",
-    "conference-2020.dateDesc": "July 23-25, 2020 | Cambridge, MA, USA",
-    "conference-2020.dateDescMore": " (with opening reception the evening of July 22)",
+    "conference-2020.dateDesc": "July 22-24, 2021 | Cambridge, MA, USA",
+    "conference-2020.dateDescMore": " (with opening reception the evening of July 21)",
     "conference-2020.locationDetails": "MIT Media Lab, Cambridge, MA",
-    "conference-2020.seeBelow": "Learn more about conference dates and locations below.",
 
     "conference-2020.date": "When:",
     "conference-2020.location": "Where:",
 
-    "conference-2020.desc1": "Join us for the Scratch Conference, an international gathering where educators, researchers, and developers share ideas for supporting creative learning with Scratch.",
+    "conference-2020.desc1": "Join us for the Scratch Conference, an international gathering where educators, researchers, and developers share ideas for supporting creative learning with Scratch. Due to the global health crisis, we have delayed the conference for one year, to July 22-24, 2021.",
```

I think the added text should be its own string so that we can easily bold it, and so that this string does not need to be retranslated:
```suggestion
    "conference-2020.desc1": "Join us for the Scratch Conference, an international gathering where educators, researchers, and developers share ideas for supporting creative learning with Scratch.",
    "conference-2020.desc1a": "Due to the global health crisis, we have delayed the conference for one year, to July 22-24, 2021.",
```

In addition to this, please add this new string to the page (at https://github.com/LLK/scratch-www/pull/3801/files#diff-63ab2bce5ecdcc31263acf95d31f7c58L31):
```jsx
                    <FormattedMessage id="conference-2020.desc1" />
                    <strong><FormattedMessage id="conference-2020.desc1a" /></strong>
```

---

Shouldn't there be a space either at the end of desc1 or at the start of desc1a so that it doesn't just look like this:

"...supporting creative learning with Scratch.Due to the global health crisis..."